### PR TITLE
Releasing on NUGET

### PR DESCRIPTION
**Pull request by ziccardi**
### Motivation

We need to publish our artifacts on nuget

JIRA: [AEROGEAR-2229](https://issues.jboss.org/browse/AGDROID-AEROGEAR-2229)

### Description

Added nuget .spec file and a document describing the process to publish an artifact

### Progress

- [x] Read documents on how to publish on nuget
- [x] Add .spec file
- [x] Add documentation on how to publish

### CODE REVIEW

File: Core/Core.nuspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>aerogear-xamarin-core</id>
+    <version>1.0.0</version>
+    <title>Aerogear Xamarin SDK Core</title>
+    <authors>RedHat</authors>
```
Comment (wojta) [inline Core/Core.nuspec]: Shouldn't be that AeroGear community? We can have other contributors in future.

**Comment by secondsun** [comment]
see https://github.com/wojta/aerogear-xamarin-sdk/pull/2 for the fix we did in PP

File: Core/Dist/make-dist.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+ * register to [NuGet central repo](https://www.nuget.org/) and generate your api key. Be sure copy them somewhere, since they won't be visualized anymore
+ * one NuGet owner of `aerogear-xamarin-core` should have added you to the list of owners to grant you access
+
+Create a branch with your new release version.
+
+2) Build the project for the desired platform:
+ 
+ * open VisualStudio, open solution
+ * choose `Debug` on the top bar schema selection select `CoreXXX` where XXX is iOS or Android
```
Comment (wei-lee) [inline Core/Dist/make-dist.md]: should this be `Release` for publishing to NuGet?

**Comment by wojta** [comment]
let's merge it, I'll try to update my Core PR with it

**Pull request merged**